tutorials/epochs/plot_10_epochs_overview.py

```diff
@@ -1,103 +1,390 @@
+#   data can be exported to a :class:`NumPy array <numpy.ndarray>` through the
+#   :meth:`~mne.Epochs.get_data` method or to a :class:`Pandas DataFrame
+#   <pandas.DataFrame>` through the :meth:`~mne.Epochs.to_data_frame` method.
+#
+# - :class:`~mne.Epochs` objects support channel selection by index or name,
+#   including :meth:`~mne.Epochs.pick`, :meth:`~mne.Epochs.pick_channels` and
+#   :meth:`~mne.Epochs.pick_types` methods.
+#
+# - :term:`SSP` projector manipulation is possible through
+#   :meth:`~mne.Epochs.add_proj`, :meth:`~mne.Epochs.del_proj`, and
+#   :meth:`~mne.Epochs.plot_projs_topomap` methods.
+#
+# - :class:`~mne.Epochs` have :meth:`~mne.Epochs.copy`,
+#   :meth:`~mne.Epochs.crop`, :meth:`~mne.Epochs.time_as_index`,
+#   :meth:`~mne.Epochs.filter`, and :meth:`~mne.Epochs.resample` methods.
+#
+# - :class:`~mne.Epochs` have :attr:`~mne.Epochs.times`,
+#   :attr:`~mne.Epochs.ch_names`, :attr:`~mne.Epochs.proj`, and :class:`info
+#   <mne.Info>` attributes.
+#
+# - :class:`~mne.Epochs` have built-in plotting methods
+#   :meth:`~mne.Epochs.plot`, :meth:`~mne.Epochs.plot_psd`,
+#   and :meth:`~mne.Epochs.plot_psd_topomap`.
+#
+#
+# Creating Epoched data from a ``Raw`` object
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# The example data we've been using thus far doesn't include pre-epoched
+# data, so in this section we'll load the continuous data and create epochs
+# based on the events recorded in the :class:`~mne.io.Raw` object's STIM
+# channels.
 
-data_path = mne.datasets.sample.data_path()
-# Load a dataset that contains events
-raw = mne.io.read_raw_fif(
-    op.join(data_path, 'MEG', 'sample', 'sample_audvis_raw.fif'))
+sample_data_folder = mne.datasets.sample.data_path()
+sample_data_raw_file = os.path.join(sample_data_folder, 'MEG', 'sample',
+                                    'sample_audvis_raw.fif')
+raw = mne.io.read_raw_fif(sample_data_raw_file, verbose=False)
 
-# If your raw object has a stim channel, you can construct an event array
-# easily
-events = mne.find_events(raw, stim_channel='STI 014')
+###############################################################################
+# As we saw in the :ref:`tut-events-vs-annotations` tutorial, we can extract an
+# events array from :class:`~mne.io.Raw` objects using :func:`mne.find_events`,
+# and we can make referencing events and pooling across event types easier if
+# we make an event dictionary:
 
-# Show the number of events (number of rows)
-print('Number of events:', len(events))
+events = mne.find_events(raw, stim_channel='STI 014')
+event_dict = {'auditory/left': 1, 'auditory/right': 2, 'visual/left': 3,
+              'visual/right': 4, 'face': 5, 'buttonpress': 32}
 
-# Show all unique event codes (3rd column)
-print('Unique event codes:', np.unique(events[:, 2]))
+###############################################################################
+# .. note::
+#
+#     We could also have loaded the events from file, using
```

Comment:
If this is a special type of file (with events previously detected from raw data and subsequently saved), it might be useful to cross-reference here instructions for saving such a file.

Reply:
added crossref to events read/write section of another tutorial